File: backend/BookstoreAPI/Models/Proveedor.cs
namespace BookstoreAPI.Models
{
    public class Proveedor
    {
        public int Id { get; set; }
        public string? Codigo { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string? RazonSocial { get; set; }
        public string? Cuit { get; set; }
        public string? Domicilio { get; set; }
        public string? Telefono { get; set; }
        public string? Mail { get; set; }
        public DateTime? FechaInhabilitacion { get; set; }
    }
}

File: frontend/bookstore-app/src/types/proveedor.ts
export interface Proveedor {
  id: number;
  codigo?: string;
  nombre: string;
  razonSocial?: string;
  cuit?: string;
  domicilio?: string;
  telefono?: string;
  mail?: string;
  fechaInhabilitacion?: string;
}

export interface CreateProveedorDto {
  nombre: string;
  razonSocial?: string;
  cuit?: string;
  domicilio?: string;
  telefono?: string;
  mail?: string;
}

export interface UpdateProveedorDto {
  nombre: string;
  razonSocial?: string;
  cuit?: string;
  domicilio?: string;
  telefono?: string;
  mail?: string;
  fechaInhabilitacion?: string;
}
